tests/blockchain/test_blockchain_transactions.py

```diff
@@ -31,9 +31,9 @@ def event_loop():
 
 
 class TestBlockchainTransactions:
-    @pytest.fixture(scope="function")
-    async def two_nodes(self):
-        async for _ in setup_two_nodes(test_constants):
+    @pytest.fixture(scope="function", params=[1, 2])
```

Comment:
If you make a single `db_version` fixture you could use it everywhere by name instead of copy/pasting the `params=`.  It would probably belong in the top level `conftest.py`.

Reply:
oh, I think I know what you mean. I'll give that a try

Reply:
I don't see how I can replace this parameterization with a separate fixture.
This is itself a fixture, the parameterized fixtures can only be used at the top level test functions, right?

Reply:
https://replit.com/@altendky/JovialThunderousIrc-1
```python
import subprocess

import pytest


@pytest.fixture(params=[1, 2])
def version(request):
    return request.param


@pytest.fixture()
def a_fixture(version):
    return version


def test(a_fixture):
    print()
    print(f"a_fixture: {a_fixture}")
    assert a_fixture in (1, 2)


# yuck
if __name__ == "__main__":
    subprocess.run(["pytest", "--capture", "no", __file__], check=True)
```

```
============================= test session starts ==============================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/runner/JovialThunderousIrc-1
collected 2 items                                                              

main.py 
a_fixture: 1
.
a_fixture: 2
.

============================== 2 passed in 0.06s ===============================
```